LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License
 
 Copyright (c) 2012 Cloud9 IDE, Inc. (Mike de Boer)
+Copyright (c) 2017 Octokit contributors
```

Comment:
@mlinksva we adopted `node-github` from the community and are renaming it to `octokit`. Does this look ok?

Reply:
2017-2018? 😁  Looks good to me.

Reply:
would `2017-present` work, too? I’ve seen that in around in other project’s license files

Reply:
`-present` is pretty bogus, it's guaranteed to be false at some point.

Reply:
However, feel free to skip the year from the added copyright line if you wish. It isn't required.